libweasyl/libweasyl/alembic/versions/a49795aa2584_add_searchtag_blacklists.py

```diff
@@ -0,0 +1,50 @@
+"""Add searchtag blacklists
+
+Revision ID: a49795aa2584
+Revises: 882fe6ace5c7
+Create Date: 2016-10-01 03:00:22.859648
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'a49795aa2584'
+down_revision = '882fe6ace5c7'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # Create the table/index for user blacklist of searchtags
+    op.create_table('searchmapuserblacklist',
```

Comment:
My first 'big unreasonable ask' is I'd like to replace all instances of 'blacklist' both in code and in site documentation with the terminology of 'restricted'. This is a better reflection of what the functionality actually means, and the codebase already has too many misleading names.